src/components/visual/FileDetail.tsx

```diff
@@ -283,6 +285,11 @@ const WrappedFileDetail: React.FC<FileDetailProps> = ({
                             <PageviewOutlinedIcon />
                           </IconButton>
                         </Tooltip>
+                        <Tooltip title={t('resubmit_file')}>
+                          <IconButton component={Link} to={{pathname: "/submit", state: {hash: file.file_info.sha256, tabContext: "2"}}}>
```

Comment:
Pass the original classification in the state so the user does not de-classify the file by accident